==== Setting Path ===== 
 
 {'ARNOLD_TESTSUITE_COMMON': '/home/krishnan/dev/mtoa/testsuite/common', 'MAYA_MODULE_PATH': '/usr/autodesk/arnold/maya2020', 'PYMEL_SKIP_MEL_INIT': '1', 'ADCLMHUB_LOG_DIR': '/home/krishnan/dev/mtoa/build/linux_x86_64/20200000/gcc_opt/testsuite/test_0182', 'MAYA_DISABLE_CIP': '1', 'ADCLMHUB_LOG_LEVEL': 'T', 'ARNOLD_MATERIALX_SEARCH_PATH': '/home/krishnan/dev/mtoa/build/linux_x86_64/20200000/gcc_opt/testsuite/test_0182', 'MAYA_RENDER_DESC_PATH': '/usr/autodesk/arnold/maya2020', 'PATH': '/usr/autodesk/arnold/maya2020/bin;/usr/autodesk/maya2020/bin;/usr/autodesk/arnold/maya2020/lib;/usr/autodesk/arnold/maya2020/plug-ins;/usr/local/bin:/opt/bin:/bin:/usr/bin:.', 'MAYA_DISABLE_CER': '1'}
 
 ==== Exec cmd ==== 
 
/usr/autodesk/maya2020/bin/Render -proj "/home/krishnan/dev/mtoa/build/linux_x86_64/20200000/gcc_opt/testsuite/test_0182" -rd "/home/krishnan/dev/mtoa/build/linux_x86_64/20200000/gcc_opt/testsuite/test_0182" -r arnold -x 160 -y 120  "/home/krishnan/dev/mtoa/build/linux_x86_64/20200000/gcc_opt/testsuite/test_0182/test.ma"
 
 
Starting "/usr/autodesk/maya2020/bin/maya"
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/usr/tmp/runtime-krishnan'
00:00:00   367MB         | log started Tue Jan 12 13:46:09 2021
00:00:00   367MB         | Arnold 6.2.0.0 [8c46d7e7] linux clang-10.0.1 oiio-2.2.1 osl-1.11.6 vdb-7.1.1 clm-2.0.0.235 rlm-12.4.2 optix-6.6.0 2021/01/12 02:56:27
00:00:00   367MB         | running on ip-192-168-0-45.eu-west-1.compute.internal, pid=143210
00:00:00   367MB         |  2 x Intel(R) Xeon(R) Gold 5118 CPU @ 2.30GHz (24 cores, 48 logical) with 63847MB
00:00:00   367MB         |  NVIDIA driver version 455.38 (Optix 60700)
00:00:00   367MB         |  GPU 0: Quadro RTX 4000 @ 1545MHz (compute 7.5) with 7982MB (7978MB available) (NVLink:0)
00:00:00   367MB         |  GPU 1: Quadro GP100 @ 1442MHz (compute 6.0) with 16278MB (15102MB available) (NVLink:0)
00:00:00   367MB         |  CentOS Linux 7 (Core), Linux kernel 3.10.0-1062.1.2.el7.x86_64
00:00:00   367MB         |  soft limit for open files raised from 4096 to 4094
00:00:00   367MB         |  
00:00:00   367MB         | loading plugins from /usr/autodesk/arnold/maya2020/plug-ins/../bin/../plugins ...
00:00:00   367MB         |  alembic_proc.so: alembic uses Arnold 6.2.0.0
00:00:00   367MB         |  cryptomatte.so: cryptomatte uses Arnold 6.2.0.0
00:00:00   367MB         |  cryptomatte.so: cryptomatte_filter uses Arnold 6.2.0.0
00:00:00   367MB         |  cryptomatte.so: cryptomatte_manifest_driver uses Arnold 6.2.0.0
00:00:00   370MB         |  usd_proc.so: usd uses Arnold 6.2.0.0
00:00:00   370MB         |  synColor_shaders.so: color_manager_syncolor uses Arnold 6.2.0.0
00:00:00   370MB         | loaded 6 plugins from 4 lib(s) in 0:00.01
Warning: file: /home/krishnan/dev/mtoa/build/linux_x86_64/20200000/gcc_opt/testsuite/test_0182/2020/prefs/filePathEditorRegistryPrefs.mel line 4: filePathEditor: Attribute 'aiImage.filename' is invalid or is not designated 'usedAsFilename'.
Warning: file: /home/krishnan/dev/mtoa/build/linux_x86_64/20200000/gcc_opt/testsuite/test_0182/2020/prefs/filePathEditorRegistryPrefs.mel line 5: filePathEditor: Attribute 'aiPhotometricLight.aiFilename' is invalid or is not designated 'usedAsFilename'.
Warning: file: /home/krishnan/dev/mtoa/build/linux_x86_64/20200000/gcc_opt/testsuite/test_0182/2020/prefs/filePathEditorRegistryPrefs.mel line 7: filePathEditor: Attribute 'aiVolume.filename' is invalid or is not designated 'usedAsFilename'.
Warning: line 1: filePathEditor: Attribute 'aiVolume.filename' and label 'VDB' have been saved already.
00:00:00   374MB WARNING |  unable to load dynamic library /usr/autodesk/arnold/maya2020/procedurals/xgenSpline_procedural.so: libAdskSeExpr.so: cannot open shared object file: No such file or directory
00:00:00   374MB WARNING |  unable to load dynamic library /usr/autodesk/arnold/maya2020/procedurals/xgen_procedural.so: libAdskSeExpr.so: cannot open shared object file: No such file or directory
Warning: line 1: filePathEditor: Attribute 'aiImage.filename' and label 'Image' have been saved already.
Warning: line 1: filePathEditor: Attribute 'aiPhotometricLight.aiFilename' and label 'IES' have been saved already.
libai.so already loaded
Successfully imported python module 'arnold'
Successfully imported python module 'mtoa'
Successfully registered renderer 'arnold'
Warning: line 1: This file is from an older version of Maya.  If saved, it will not be readable by previous versions.
File read in  4.1 seconds.
Result: /home/krishnan/dev/mtoa/build/linux_x86_64/20200000/gcc_opt/testsuite/test_0182/test.ma
00:00:05   561MB WARNING | rendering with watermarks because of failed authorization:
00:00:05   561MB         |  [rlm] error initializing license system:
00:00:05   561MB         |  [rlm]  * Can't read license data (-102)
00:00:05   561MB         |  [clm.v1] generic license checkout error (22)
00:00:05   561MB         |  [clm.v2] SERVICE (4): this product has not been registered with the licensing service yet
00:00:05   561MB         |  environment variables:
00:00:05   561MB         |         ARNOLD_LICENSE_ORDER   = (not set)
00:00:05   561MB         |         ARNOLD_LICENSE_MANAGER = (not set)
00:00:05   561MB         |  [rlm]  solidangle_LICENSE     = (not set)
00:00:05   561MB         |  [rlm]  RLM_LICENSE            = (not set)
00:00:05   561MB         |  [clm]  ADSKFLEX_LICENSE_FILE  = (not set)
00:00:05   561MB         |  [clm]  LM_LICENSE_FILE        = (not set)
Scene /home/krishnan/dev/mtoa/build/linux_x86_64/20200000/gcc_opt/testsuite/test_0182/test.ma completed.
